### DISCUSSION TITLE
How to add instances

### DISCUSSION THREAD

**Discussion by dennisa72295** [trimble-oss/dba-dash#558] [Q&A] [answered]
I'm just getting started with DBA Dash and I can't seem to figure out how to add any more instances.  In my environment we don't use the default port of 1433, so every instance is listening on a different port.  There can be more than one instance on a given server.  When I try to add and instance as servername,port it doesn't show up in the dashboard.  What is the proper way to add additional instances when they are on different ports?
I figured it out.  I had failed to add the service account as a login to all the remote instances.  All good with this.

**Comment by DavidWiseman** [comment, marked as answer]
Great.  Glad you got it sorted.
